package.json

```diff
@@ -148,7 +149,7 @@
     "jszip": "3.5.0",
     "lodash": "4.17.19",
     "md5.js": "1.3.5",
-    "monaco-editor-webpack-plugin": "1.9.0",
+    "monaco-editor-webpack-plugin": "^2.1.0",
```

Comment:
did we intentionally start using compatibility versions?

my experience has taught me this is a lie, that no version upgrade should be considered compatible or safe

Reply:
I removed them from this PR, if we want to take them out of the file generally let's do that in a different PR